src/components/inputs/upload-input-v2/dropzone.js

```diff
@@ -15,6 +16,8 @@ export class DropzoneJS extends React.Component {
         super(props);
         this.state = {files: []};
         this.onUploadComplete = this.onUploadComplete.bind(this);
+
+        this.accessToken = null;
```

Comment:
this does not serve its porpoises
we need to get the token on each chunk upload otherwise we could have an stale token